itest/lnd_coop_close_external_delivery_test.go

```diff
@@ -58,11 +63,8 @@ func testCoopCloseWithExternalDelivery(ht *lntest.HarnessTest) {
 // not. Some users set this value to be an address in a different wallet and
 // this should not affect our ability to accurately report the settled balance.
 func testCoopCloseWithExternalDeliveryImpl(ht *lntest.HarnessTest,
-	upfrontShutdown bool, deliveryAddressType lnrpc.AddressType) {
-
-	alice := ht.NewNodeWithCoins("Alice", nil)
-	bob := ht.NewNodeWithCoins("bob", nil)
-	ht.ConnectNodes(alice, bob)
+	alice, bob *node.HarnessNode, upfrontShutdown bool,
```

Comment:
I think it's better to start new nodes to keep each test independent. We can actually do sth similar to 
https://github.com/lightningnetwork/lnd/blob/c9fe051b2f90fe4c03c9d9891b44805a5112db66/itest/lnd_wallet_import_test.go#L32

So sth like below, which keeps blocks mined small, also nice for the itest scheduler to load balance.
```go
var coopCloseWithExternalTestCases = []*lntest.TestCase{
	{
		Name: "set P2WPKH delivery address at open",
		TestFunc: func(ht *lntest.HarnessTest) {
			testCoopCloseWithExternalDeliveryImpl(
				ht, true, false,
				lnrpc.AddressType_UNUSED_WITNESS_PUBKEY_HASH,
			)
		},
	},
	{
		Name: "set P2WPKH delivery address at close",
		TestFunc: func(t *lntest.HarnessTest){
			...
		},
	},
        ...
}
```